src/hal/components/invert.comp

```diff
@@ -25,6 +25,7 @@ param rw float deadband "The \\fBout\\fR will be zero if \\fBin\\fR is between -
 function _;
 license "GPL";
 author "Stephen Wille Padnos";
+see_also " invert, div2 ";
```

Comment:
This should be see_also " invert(9), div2(9) ";

Also, perhaps sort the list alphabetically?